src/Makevars

```diff
@@ -1,15 +1,15 @@
 # Calls to 'firstword' are used to strip out
 # a possible '-m64' attached to the CC variable
 # (or other compiler arguments)
-PKG_CPPFLAGS += $(CXX1XSTD) -I../inst/include/
+PKG_CPPFLAGS += $(CXX11STD) -I../inst/include/
```

Comment:
Does R 3.0.2 understand `CXX11STD`? (Just because we in theory support R back to that version)

Reply:
Hm, I thought that "CXX1X" had been deprecated in favor of "CXX11"?

[From the manual:](https://cran.r-project.org/doc/manuals/r-release/R-admin.html#Customizing-package-compilation)

> R versions 3.1.0 to 3.3.3 used CXX1X rather than CXX11, and these forms are deprecated but still accepted.

Reply:
Indeed it's been deprecated, but because RcppParallel currently requires R >= 3.0.2, I just wanted to make sure that the package still builds successfully there.

Alternatively, we can just bump the minimum R version in the DESCRIPTION to R 3.1.0?

Reply:
Wasn't the CXX1X != CXX11 change done in R 3.4.0?   IIRC it still understands the older form but prefers the newer.